frontend/src/components/table/Table.js

```diff
@@ -99,12 +102,24 @@ export default class Table extends PureComponent {
     return arrayIndex.slice(selectedArr[0], selectedArr[1] + 1);
   };
 
+  /**
+   * @summary Updates the start reference used for the multi selection when SHIFT + arrow up/down keys are pressed
+   * @param {*} currentId - the current index in the array of rows
+   */
+  updateMultiSelStart = (currentId) => {
+    this.multiSelStart =
+      !this.multiSelStart && this.multiSelStart !== 0
```

Comment:
pls simply this logic 
and i suggest to inline the function.

Reply:
do you mean     `this.multiSelectionStartIdx = this.multiSelectionStartIdx || currentId; ` ? This is not going to work.
Reason: https://www.loom.com/share/c46b280c4c8046fba0a0c0d537506e35